contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -202,33 +203,58 @@ export default function Feedback() {
 
 ## Capture survey responses
 
-The main event you must capture to track survey results is `survey_sent` which we've already shown. It is formatted like this:
+The main event you must capture to track survey results is `survey sent`. PostHog supports two formats for survey responses, but we strongly recommend using ID-based responses as they are more reliable and resistant to changes in question ordering:
 
 ```js-web
 posthog.capture("survey sent", {
     $survey_id: survey.id, // required
-    $survey_response: survey_response // required. `survey_response` must be a text value.
+    $survey_response_question_id: survey_response // ID-based (recommended)
```

Comment:
@lucasheriques i think we should improve this, people might think this is literally `$survey_response_question_id` instead of `$survey_response_{question_ID}`, I'd even add the key as an example eg `$survey_response_550e8400-e29b-41d4-a716-446655440000`

Reply:
more importantly, the docs do not tell how to get the corresponding id (eg 550e8400-e29b-41d4-a716-446655440000)